### DISCUSSION TITLE
Reconstruction of parent phase austenite for a certain phase in multi-phase steel

### DISCUSSION THREAD

**Discussion by Mikewangkang** [AzdiarGazder/mtexTools#8] [Q&A] [answered]
Hi， @AzdiarGazder 
Based on the script you developed, ferriteQuantifier/demo_ferriteQuantifier_multiPhaseMaps.m, I implemented the phase classification. I want to reconstruct the parent phase austenite separately for the bainite, packet reconstruction and ellipse fitting of the bainite grains. But I don't know how to edit the script, can you help me?
Best wishes
Wangkang

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @Mikewangkang

To answer your question:
After you have completed segmentation, please save the ebsd1 variable as a new *.CTF file.
```matlab
ebsd1.export('myNewFileName.ctf')
```
Following that, go to ORTools and load the *.CTF file.
Depending on the phase names, choose "Bainite" as the child phase and "Iron fcc" as the parent phase to begin reconstruction.

<ins>**Lastly, and importantly, if this explanation resolves your new issue, please mark this message as your answer as well.**</ins>

Warm regards,
Azdi
